tests/bluetooth/shell/boards/native_posix.conf

```diff
@@ -0,0 +1,10 @@
+CONFIG_LOG_MODE_IMMEDIATE=y
```

Comment:
This should be the default in native_posix (no harm done having it here though)

Reply:
Removed this line as suggested.